### PR TITLE
Rename Subscription -> Handle

### PR DESCRIPTION
**Pull request by willscott**
two changes:
1. move simple-centralized to a future load testing benchmark.
This is mainly to get us to a place where everything compiles :shipit: (allowing to tools to then do automatic refactoring on the project)

2. using that refactoring, rename subscription to handle in libtalek

### CODE REVIEW

File: benchmark/client/client.go [hunk 1]
```diff
@@ -56,8 +56,8 @@ func main() {
 	//log.Println("Created c1")
 
 	handle, _ := libtalek.NewTopic()
```
Comment (ryscheng) [inline benchmark/client/client.go]: seems minor, but shouldn't NewTopic generate a `topic`?

**Comment by ryscheng** [comment]
Just checking that we don't use simple-distributed for anything anymore right?
I see it was deleted.

**Comment by willscott** [comment]
simple-distributed sure looked like basically the same sketch as simple-centralized, but was using a broken code path (afaict).

File: benchmark/client/client.go (hunk 1 above)
Comment (willscott) [inline benchmark/client/client.go]: :+1:

**Comment by ryscheng** [comment]
:+1:

**Pull request merged**